### PR TITLE
Xamarin Templates - Localization + Web View

### PR DESCRIPTION
**Pull request by milazzom**
This PR includes:

* Adds Localization support for Xamarin.Forms projects using the CodeBehind, MVVMBasic, PrismSharedProject, and PrismNetStd frameworks.
* Edits .csproj and .projitems files in the base project templates to support adding custom build actions from postaction files.
* Adds Xamarin.Forms web view page for all the supported STS Xamarin frameworks.
* Adds Param_ProjectName property for templates added to the current generation session when referenced as a dependency from another template.

### CODE REVIEW

**Review by jermws — COMMENTED** [review]
Why are we checking in App 19 code?

File: samples/localization/SharedProject/LocalizationTest/LocalizationTest/LocalizationTest/Services/Localizer.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+using System.Text;
+
+using Xamarin.Forms;
+
+[assembly: Dependency(typeof(LocalizationTest.Services.Localizer))]
+namespace LocalizationTest.Services
+{
+    public class Localizer : ILocalizer
+    {
+
```
Comment (jermws) [inline samples/localization/SharedProject/LocalizationTest/LocalizationTest/LocalizationTest/Services/Localizer.cs]: Remove blank line in the template

Comment (milazzom) [reply samples/localization/SharedProject/LocalizationTest/LocalizationTest/LocalizationTest/Services/Localizer.cs]: I'm going to remove this particular sample solution (shared project) as it was added for the Xamarin team (David and Pierce) to provide some feedback on.

Comment (milazzom) [reply samples/localization/SharedProject/LocalizationTest/LocalizationTest/LocalizationTest/Services/Localizer.cs]: This one isn't part of the template.  The .NetStd version of the sample project is functional and can be used to demonstrate how to use the capability in addition to an example page template.

**Review by jermws — APPROVED** [review]

**Review by jermws — APPROVED** [review]

**Pull request merged**